### ISSUE TITLE
Update labels in mdn/content

### ISSUE THREAD

**Issue by bsmth** [mdn/mdn#287]
### Labels to add

- [x] Common labels (all lower case no hyphens)

  - [x] `needs triage` Triage needed by staff and/or partners. Automatically applied when an issue is opened.
  - [x] `good first issue` A good issue for newcomers to get started with.
  - [x] `help wanted` If you know something about this topic, we would love your help!
  - [x] `needs info` Needs more information to review or act on.
  - [x] `expert help needed` Needs more information from a subject matter expert (SME).
  - [x] `idle` Issues and pull requests with no activity for three months.
  - [x]  `on hold` Waiting on something else before this can be moved forward.
  - [x]  `wontfix` Outside the scope of the project or requires a significant amount of time and resources to fix.
  - [x]  `for later` Not planned at this time.
  - <strike>`needs content update` Needs update to the content repo to support this change.</strike>
  - [x]  `bug` Indicates an unexpected problem or unintended behavior.
  - [x]  `chore` A routine task.
  - [x]  `enhancement` Improves an existing feature.

#### Effort / size of task

- [x] `effort: small` This task is a small effort.
- [x] `effort: medium` This task is a medium effort.
- [x] `effort: large` This task is large effort.

#### Priority

- [x] `p0` Urgent. We will address this as soon as possible.
- [x] `p1` We will address this soon and will provide capacity from our team for it in the next few releases.
- [x] `p2` We want to address this but may have other higher priority items.
- [x] `p3` We don't have visibility when this will be addressed.
